consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/IClaimStatusUpdate.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.moj.sdt.cmc.consumers.api;
+
+import uk.gov.moj.sdt.cmc.consumers.model.ClaimStatusUpdateRequest;
+
+public interface IClaimStatusUpdate {
+
+    Object claimStatusUpdate(ClaimStatusUpdateRequest claimStatusUpdateRequestObj, String idAmId, String sdtRequestId);
```

Comment:
can we use the method signature as bellow:
claimStatusUpdate(String idAmId, String sdtRequestId, ClaimStatusUpdateRequest claimStatusUpdateRequestObj)